compatibility_lib/compatibility_lib/fake_compatibility_store.py

```diff
@@ -95,6 +96,32 @@ def get_pair_compatibility(self, packages: List[package.Package]) -> \
                 frozenset(packages),
                 []))
 
+    def get_pairwise_compatibility_for_package(self, package_name: str) -> \
```

Comment:
Put this in a separate PR and submit it first.

Reply:
I submitted another PR just now.